selfdrive/car/tests/test_fw_fingerprint.py

```diff
@@ -67,7 +68,7 @@ def test_blacklisted_ecus(self):
     blacklisted_addrs = (0x7c4, 0x7d0)  # includes A/C ecu and an unknown ecu
     for car_model, ecus in FW_VERSIONS.items():
       with self.subTest(car_model=car_model):
-        CP = interfaces[car_model][0].get_params(car_model)
+        CP = interfaces[car_model][0].get_params(car_model, gen_empty_fingerprint(), [])
```

Comment:
let's make a helper for this. name of the helper should make it obvious that the carParams are potentially wrong or incomplete